internal/api/resources_test.go

```diff
@@ -404,13 +405,42 @@ func TestPutResource(baseT *testing.T) {
 	})
 }
 
+func TestMaxAssetSizeFor(t *testing.T) {
+	var (
+		maxBarSize = p2uint64(42)
+		maxFooSize = p2uint64(30)
+	)
+
+	cfg := core.Config{
+		MaxAssetSizeRules: []core.MaxAssetSizeRule{
+			{
+				Regex: regexp.MustCompile("^foo.*$"),
+				Value: maxFooSize,
+			},
+			{
+				Regex: regexp.MustCompile("^.*bar$"),
```

Comment:
`^$` is only added in main.go, so when testing I need to add it here, too. Should I move that to also apply to regex's that are not read from the config file?

Reply:
A nice solution would be to move the whole parsing of the MAX_ASSET_SIZES string into a public function. Then we could initialize the MaxAssetSizeRules as `ParseMaxAssetSizeRules("foo=30,bar=42")` or such.